marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -586,7 +587,7 @@ function validatePropertyMapping(fullMapping, userParameterNames, propertyName,
     // Thus, this is not using es.mappingCompile, which does validation, and just invokes the transform instead.
     validateXmlMapping(xmlMapping);
   } catch (e) {
-    return hubUtils.getErrorMessage(e);
+    return mappingLibrary.extractErrorMessageForMappingUI(e);
```

Comment:
We really need to do "mappingLib" here to be consistent with e.g. entity-lib.sjs. So for mapping/default/lib.sjs, let's call that "mappingStepLib".